### ISSUE TITLE
Malware- Unable to download new Patch

### ISSUE THREAD

**Issue by tumadreee** [finndev/PokeBuddy#551]
My Pokebuddy.exe had an error and disappeared. Now the new update 15 minutes ago is being blocked for malware. 
Why the sudden change?

**Comment by curtiola** [comment]
Older versions have been flagged as well by Antivirus, I assume it's a false positive, finndev ain't no virus dev haha

**Comment by tumadreee** [comment]
Was this a forced updated? I'm still wondering why my last version just error and disappeared

**Comment by curtiola** [comment]
Yeah, when you open it after a new version is pushed it will auto update

**Comment by finndev** [comment]
Please tell me what AV you use and I'll ask for a whitelist, altho files are already signed with a digital signature
